src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/XamarinFormsMapsApplicationProject.cs

```diff
@@ -17,19 +17,13 @@ static XamarinFormsMapsApplicationProject ()
 		public XamarinFormsMapsApplicationProject ([CallerMemberName] string packageName = "")
 			: base (packageName: packageName)
 		{
-			if (Builder.UseDotNet) {
-				PackageReferences.Add (KnownPackages.XamarinFormsMaps_4_7_0_1142);
-				PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Base);
-				PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Basement);
-				PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Maps);
-				PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Tasks);
-				PackageReferences.Add (KnownPackages.Xamarin_Build_Download);
+      PackageReferences.Add (KnownPackages.XamarinFormsMaps_4_7_0_1142);
+      PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Base);
+      PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Basement);
+      PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Maps);
+      PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Tasks);
+      PackageReferences.Add (KnownPackages.Xamarin_Build_Download);
```

Comment:
This is no big deal, but tabs vs spaces:
```suggestion
			PackageReferences.Add (KnownPackages.XamarinFormsMaps_4_7_0_1142);
			PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Base);
			PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Basement);
			PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Maps);
			PackageReferences.Add (KnownPackages.Xamarin_GooglePlayServices_Tasks);
			PackageReferences.Add (KnownPackages.Xamarin_Build_Download);
```

Reply:
Thanks, fixed